transformers/events/dent/transformer_test.go

```diff
@@ -43,10 +43,18 @@ var _ = Describe("Dent Transformer", func() {
 		models, err := transformer.ToModels(constants.FlipABI(), []core.EventLog{test_data.DentEventLog}, db)
 		Expect(err).NotTo(HaveOccurred())
 
+		var msgSenderId int64
+		msgSender := common.HexToAddress(test_data.DentEventLog.Log.Topics[1].Hex()).Hex()
+		msgSenderErr := db.Get(&msgSenderId, `SELECT id FROM public.addresses WHERE address = $1`, msgSender)
```

Comment:
This must be a pattern in our older transformers because I had similar code in my test. @rmulhol had suggested trying to get rid of the raw SQL look up here and use `shared.GetOrCreateAddress`